packages/payload/src/queues/utilities/getJobTaskStatus.ts

```diff
@@ -0,0 +1,38 @@
+import type { TaskConfig, TaskType } from '../config/types/taskTypes.js'
+import type { BaseJob, JobTaskStatus } from '../config/types/workflowTypes.js'
+
+type Args = {
+  jobLog: BaseJob['log']
+  tasksConfig: TaskConfig<TaskType>[]
+}
+
+export const getJobTaskStatus = ({ jobLog }: Args): JobTaskStatus => {
+  const taskStatus: JobTaskStatus = {}
```

Comment:
surprising that this is not Bitchin' about any missing properties (`taskSlug`, `totalTried` maybe should be instantiated to `0`)

Reply:
It's mapped from taskSlug => taskID => actual task status. Those properties are already required, but the JobTaskStatus type itself can still be empty as there may be 0 tasks